src/main/java/org/intellij/sdk/codesync/codeSyncSetup/CodeSyncSetup.java

```diff
@@ -501,11 +502,15 @@ public static boolean uploadRepo(String repoPath, String repoName, String[] file
         JSONObject payload = new JSONObject();
         boolean isPublic = false;
         if (!isSyncingBranch) {
-            isPublic = CodeSyncMessages.showYesNoMessage(
-                    Notification.PUBLIC_OR_PRIVATE,
-                    Notification.PUBLIC_OR_PRIVATE,
-                    project
+//            isPublic = CodeSyncMessages.showYesNoMessage(
+//                    Notification.PUBLIC_OR_PRIVATE,
+//                    Notification.PUBLIC_OR_PRIVATE,
+//                    project
+//            );
+            RepoPublicPrivateDialog repoPublicPrivateDialog = new RepoPublicPrivateDialog(
+                    Notification.PUBLIC_OR_PRIVATE_TITLE, Notification.PUBLIC_OR_PRIVATE, project
```

Comment:
I think it would be better to move these inside the class. We should only take arguments which are dynamic and in this case the messages are constants.